pallets/subbridge/src/chainbridge.rs

```diff
@@ -1204,9 +1224,19 @@ pub mod pallet {
 						MultiLocation::new(
 							0,
 							X3(
-								GeneralKey(b"cb".to_vec().try_into().expect("less than length limit; qed")),
+								GeneralKey(
+									b"cb"
+										.to_vec()
+										.try_into()
+										.expect("less than length limit; qed")
```

Comment:
I suggest to make a function takes `&u8` as the input and `BoundedVec<u8>` ad the output to simplify the test code.

Reply:
Consider it will return a `Result` and we always need to handle return value, sounds will not bring too many conveniences

Reply:
Implemented like this by introducing WrapSlice to convert &[u8] to bounded vec

Reply:
Fantastic!